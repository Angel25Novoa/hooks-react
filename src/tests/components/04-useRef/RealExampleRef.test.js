import React from 'react'
import {shallow} from 'enzyme'
import RealExampleRef from '../../../components/04-useRef/RealExampRef'

describe('Pruebas en RealExampleRef', () => {
  //Debe mostrarse correctamente
  const wrapper = shallow(<RealExampleRef/>)
  test('Debe mostrarse correctamente', () => {
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false)
  })
  //Debe de mostrar la información
  test('Debe de mostrar la información', () => {
    wrapper.find('button').simulate('click')
    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true)
  })
})