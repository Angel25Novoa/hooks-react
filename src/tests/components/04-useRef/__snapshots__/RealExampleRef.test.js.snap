// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en RealExampleRef Debe mostrarse correctamente 1`] = `
<div>
  <h1>
    RealExampRef
  </h1>
  <hr />
  <button
    className="btn btn-info"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</div>
`;
